modules/file-system/filestore/variables.tf

```diff
@@ -96,10 +96,6 @@ variable "filestore_tier" {
     ], var.filestore_tier)
     error_message = "Allowed values for filestore_tier are 'BASIC_HDD','BASIC_SSD','HIGH_SCALE_SSD','ZONAL','ENTERPRISE'.\nhttps://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/filestore_instance#tier\nhttps://cloud.google.com/filestore/docs/reference/rest/v1beta1/Tier."
```

Comment:
Remove reference to the legacy HIGH_SCALE_SSD tier.

https://cloud.google.com/filestore/docs/service-tiers#legacy-tiers